### PR TITLE
Add sign and verify tests for Sign1 w/ PS256

### PR DESCRIPTION
**Pull request by thomas-fossati**
Partially addresses #1

### CODE REVIEW

**Review by henkbirkholz — APPROVED** [review]

File: sign1-sign-0004.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+      "cborDiag": "18([h'A1013824', {4: 'bilbo.baggins@hobbiton.example'}, h'546869732069732074686520636F6E74656E742E', h'788EF717E37B7A127010F87D458BE5788151A734AEAF1EF52A8F7AC40D05DAF1F5DC575B29C0BF0E1D326CFCAE1AF5DCF62B6DC26EB18A35456FD7C3477774D2C664BABC5DB7618309DC2BA38392C3DBA61F0D27974EA5D4DF329060334B8BBD9D8F41CC090913CB2CD4470AE2C8560173793E703D2DDA52A4E804ECECD57DB5BA30200B4D0939F89ADC1F13F829E516625109684B429A088E0A2766564CFD2EE1A1ACC4F20AD981E4BC27E427D754481CA93EE16A7677E24CDD31D03B44FE3260EE35BBB5B57C1264AF6CA879F67B39B2680D920252E37B916E970BAA4762C65CC4EA071B2AB65C5C992518597C7B7FBF9608564C5C2C3CAAE9449D46F8DF32'])"
+    },
+    "fixedOutputLength": 65
```
Comment (qmuntal) [inline sign1-sign-0004.json:37]: RSA is deterministic so we should be able to compare against the full output, no?

Comment (thomas-fossati) [reply sign1-sign-0004.json:37]: actually PSS is probabilistic -- randomness comes from the salt parameter, which can be set to 0 to get a deterministic output.

**Review by SteveLasker — APPROVED** [review]
LGTM

File: sign1-sign-0004.json (hunk 1 above)
Comment (qmuntal) [reply sign1-sign-0004.json:37]: You are right, I forgot we were talking about RSA-PSS.

**Comment by qmuntal** [comment]
LGTM. Successfully used the new test vectors in https://github.com/veraison/go-cose/pull/21

**Review by qmuntal — APPROVED** [review]

**Comment by thomas-fossati** [comment]
> LGTM. Successfully used the new test vectors in [veraison/go-cose#21](https://github.com/veraison/go-cose/pull/21)

👍

**Pull request merged**